### PR TITLE
Added an about and a work section to the homepage of this project 

### PR DESCRIPTION
**Pull request by AyoMoses1**
I added two more sections to the homepage of this project which is highlighted in the title above. Best practices were followed to ensure smooth compatibility with the previous merge request. Please feel free to go through these newly added sections and drop your recommendations where necessary

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -83,3 +104,149 @@ ul.socials {
...
+
+ul.works_stack li {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 4px 12px;
+  background: #ebebff;
+  border-radius: 8px;
+  margin-right: 8px;
+}
```
Comment (Ugboaja-Uchechi) [inline styles.css]: - [ ] Great job with your design 👏. Kindly give your list the correct `color` as seen in your chosen Figma design 🙏 

Figma Design | Your Design
------------- | -------------
image  |  image

File: styles.css [hunk 2]
```diff
@@ -83,3 +104,149 @@ ul.socials {
...
 }
+
+ul.works_grid {
+  display: grid;
+  grid-template-columns: 1fr;
+  padding: 114px 24px;
+  grid-row-gap: 88px;
+  list-style-type: none;
+}
+
```
Comment (Ugboaja-Uchechi) [inline styles.css:119]: - [ ] Great job with your design 💪. As seen in your chosen Figma design kindly give your work section a `background-color` 🙏

Figma Design | Your Design
------------- | -------------
image  |  image

File: styles.css [hunk 3]
```diff
@@ -83,3 +104,149 @@ ul.socials {
...
+  background: #6070ff;
+  box-shadow: 0 8px 16px rgba(64, 83, 252, 0.24);
+  color: #fff;
+}
+
+#about {
+  display: flex;
+  flex-direction: column;
+  padding: 114px 24px;
+}
```
Comment (Ugboaja-Uchechi) [inline styles.css]: - [ ] Kindly give your about section a `border` and a `border-top-right-radius` as seen in your chosen Figma design 🙏 

Figma Design | Your Design
------------- | -------------
image  |  image

**Review by Ugboaja-Uchechi — CHANGES_REQUESTED** [review]
Hi @AyoMoses1, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- Linters are passing ✔️ 
- README is descriptive 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 2 above)
Comment (AyoMoses1) [reply styles.css:119]: Thanks for your feedback @Ugboaja-Uchechi. However, I've checked the Figma file and I can't seem to locate the background color for the work section.

Comment (AyoMoses1) [reply styles.css:119]: Hi @Ugboaja-Uchechi . About this, I just noticed it wasn't stated in the Figma file I duplicated in my drafts. However, it is on the original file. Thanks once again for highlighting this

File: styles.css (hunk 3 above)
Comment (AyoMoses1) [reply styles.css]: Thanks @Ugboaja-Uchechi. Please as regards this, I noticed there was no border property assigned to the about section from the Figma file. And also for the border-top-right-radius, I only assumed a value of 100px as I can't seem to find it on the Figma file

Comment (Ugboaja-Uchechi) [reply styles.css]: Hi @AyoMoses1, not all style properties are written in the Figma design that's why I stated the style properties for you so you can get a better idea. I hope this clears it up for you 😊

**Review by christianbiring1 — APPROVED** [review]
Hi @AyoMoses1 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# STATUS: APPROVED :tada: 
nicee

## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Descri[tive PR `title` and `summary` :+1: 
- [ ] Good looking UI :art: 

## Optional suggestions
- [ ] `When you get feedback from the reviewer kindly double checks what was required and always have an opening spirit to criticism and feedback.`
`Happy coding` :rocket: 
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**